src/components/notifications/PageNotifications.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <TmPage data-title="My alerts" hide-header>
+    <div class="header">
+      <h1>Notifications</h1>
+    </div>
+
+    <TmDataMsg
+      v-if="notifications.length === 0"
+      icon="error"
+      icon-color="var(--dark-grey-blue)"
+    >
+      <div>
+        <i class="material-icons nontranslate">error</i>
+      </div>
+      <div slot="title">You don't have any notifications yet</div>
+      <div slot="subtitle">Don't worry, they are on their way!</div>
+    </TmDataMsg>
+
+    <div
+      v-for="notification in notifications"
+      :key="notification.id"
+      class="notification"
+    >
+      <div class="content">
+        <img :src="notification.icon" />
+        <div>
+          <h3 class="title">{{ notification.title }}</h3>
+        </div>
+      </div>
+      <i class="material-icons notranslate">chevron_right</i>
+    </div>
+  </TmPage>
+</template>
+
+<script>
+import TmPage from "../common/TmPage"
+import TmDataMsg from "common/TmDataMsg"
+import { mapGetters } from "vuex"
+import gql from "graphql-tag"
+
+export default {
+  name: "PageNotifications",
+  components: {
+    TmPage,
+    TmDataMsg
+  },
+  data: () => ({
+    notifications: []
+  }),
+  computed: {
+    ...mapGetters([`networks`]),
+    networkIds() {
+      return this.networks.map(network => network.id)
+    },
+    allSessionAddresses() {
```

Comment:
move into store?

Reply:
Well, actually there is something about this I wanted to discuss with you all, about the architecture. Right now we are just getting the accounts from sessions, but there is also another Item much more complete in localStorage: `cosmos-wallets`. Those are all the wallets that the user has used since the beginning.

The only issue is that Ledger sessions /wallets don't get stored there. But we could change that easily. 

Anyway, that would be another PR

Reply:
> cosmos-wallets

those only store local keys